runtime/compiler/env/j9jit.tdf

```diff
@@ -81,5 +81,5 @@ TraceEvent=Trc_JIT_OverallCompCPU         Group=perfmon Overhead=1 Level=5 Test
 
 TraceEvent=Trc_JIT_SCCInfo                Group=perfmon Overhead=1 Level=5 Test Template="SCC: name=%s path=%s size=%u free=%u softMax=%u ROMClass=%u AOTCode=%u AOTData=%u JITHint=%u JITProfile=%u ROMClasses=%u AOTMethods=%u disabledReason=%d"
 
-TraceEvent=Trc_JIT_IProfiler_unrecognized Overhead=1 Level=1 Test Template="Unrecognized bytecode (pc=%p, bc=%d) cursor %p in buffer %p of size %d"
+TraceEvent=Trc_JIT_IProfiler_unrecognized Overhead=1 Level=1 Test Template="Unrecognized bytecode (pc=%p, bc=%u) cursor %p in buffer %p of size %zu"
```

Comment:
Actually, this should be TraceException. Which uses a different circular buffer so the Exceptions don't get flushed as quickly as Events.

Reply:
I can change it but I don't think it matters as this tracepoint is used to report bogus data before we assert and take the VM down.

Reply:
Updated to use Exception instead